src/transaction/log_tran_table.c

```diff
@@ -1588,6 +1588,14 @@ logtb_clear_tdes (THREAD_ENTRY * thread_p, LOG_TDES * tdes)
   LSA_SET_NULL (&tdes->rcv.atomic_sysop_start_lsa);
   LSA_SET_NULL (&tdes->rcv.analysis_last_aborted_sysop_lsa);
   LSA_SET_NULL (&tdes->rcv.analysis_last_aborted_sysop_start_lsa);
+
+  /* reflect the session (or prm_Def) */
+  tdes->isolation = (TRAN_ISOLATION) prm_get_integer_value (PRM_ID_LOG_ISOLATION_LEVEL);
+  tdes->wait_msecs = prm_get_integer_value (PRM_ID_LK_TIMEOUT);
```

Comment:
how about using `PRM_ID_LK_TIMEOUT_SECS` instead of `PRM_ID_LK_TIMEOUT` ? 

In other codes, only `PRM_ID_LK_TIMEOUT_SECS` is used. 
When the `PRM_ID_LK_TIMEOUT` parameter is set by the user, the value is converted into `second` units. This value is also used for `PRM_ID_LK_TIMEOUT_SECS`. If you use this parameter, user can clearly understand that the unit of the timeout value is `second`.

Reply:
@joohok In my opinion, `PRM_ID_LK_TIMEOUT ` is better now because `PRM_ID_LK_TIMEOUT_SECS` is deprecated.

Reply:
Or, is `PRM_DEPRECATED` just to prevent users from using it, not in the code?

Reply:
> Or, is `PRM_DEPRECATED` just to prevent users from using it, not in the code?

Maybe this is right. Because  only the `PRM_ID_LK_TIMEOUT_SECS` is used in the code when set the lock timeout in client initialization step (boot_initialize_client ()) , but it is not opened to the user.